pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -28,4 +35,10 @@ func (pp *ProxyProcessor) OnInvokeEnd(endDetails *proxy.InvocationEndDetails) {
 	log.Debug("[proxy] Invocation has finished at :", endDetails.EndTime)
 	log.Debug("[proxy] Invocation isError is :", endDetails.IsError)
 	log.Debug("[proxy] ---------------------------------------")
+
+	if endDetails.IsError {
+		serverlessMetrics.SendErrorsEnhancedMetric(
+			pp.Tags, time.Now(), pp.MetricsChannel,
```

Comment:
I think using endDetails.endTime instead of time.Now() might be more accurate?

Reply:
This is where the floating point issue was coming from I bet.